tests/unit/dataset/dataset_test.py

```diff
@@ -1952,7 +1952,7 @@ def test_load_no_files_precomputed_raises_exception(precomputed_dataset_configur
     shutil.rmtree(dataset.paths.precomputed_events, ignore_errors=True)
     dataset.paths.precomputed_events.mkdir()
 
-    with pytest.raises(RuntimeError):
+    with pytest.raises(FileNotFoundError):
```

Comment:
I am very confused why the exception changed, but I think `FileNotFoundError` also makes more sense for this test. :man_shrugging:

Reply:
i made a mistake when raising the runtimeerror during scan and reverted the change of the test.